packages/lwc/scripts/utils/rollup.js

```diff
@@ -83,8 +86,20 @@ async function generateTarget({ inputOptions, outputOptions, display }) {
         `pid: ${process.pid}`.padEnd(10, ' '),
     ].join(' | ');
 
-    const bundle = await rollup(inputOptions);
-    await bundle.write(outputOptions);
+    if (minify) {
+        // We don't use rollup-plugin-terser because we want to minify the whole file at the end,
+        // which is more efficient than doing it as part of the input pipeline which would require
+        // minifying the same source file multiple times.
```

Comment:
It's possible to do the same thing with rollup plugin as well. The `output` also accept plugins: https://github.com/TrySound/rollup-plugin-terser#using-as-output-plugin

Reply:
TIL. Switched to using the `output` plugin, and the Rollup code is simpler and still generates the same artifacts. And still faster. 🙂